src/main/java/org/gitlab4j/api/IssuesApi.java

```diff
@@ -621,4 +622,52 @@ public TimeStats getTimeTrackingStats(Integer projectId, Integer issueIid) throw
             return (GitLabApi.createOptionalFromException(glae));
         }
     }
+
+    /**
+     * Get list containing all the merge requests that will close issue when merged.
+     *
+     * GET /projects/:id/issues/:issue_iid/closed_by
+     *
+     * @param projectId the project ID that owns the issue
+     * @param issueIid the internal ID of a project's issue
+     * @return a List containing all the merge requests what will close the issue when merged.
+     * @throws GitLabApiException if any exception occurs
+     */
```

Comment:
We have adopted using ```Object projectIdOrPath``` with new methods, this allows a project ID, project path, or actual Project instance to be passed in.  Please change these 3 methods as follows:
```
public List<MergeRequest> getClosedByMergeRequests(Object projectIdOrPath, Integer issueIid) throws GitLabApiException {
    return (getClosedByMergeRequests(projectIdOrPath, issueIid, 1, getDefaultPerPage()));
}

public List<MergeRequest> getClosedByMergeRequests(Object projectIdOrPath, Integer issueIid, int page, int perPage) throws GitLabApiException {
    Response response = get(Response.Status.OK, getPageQueryParams(page, perPage),
            "projects", getProjectIdOrPath(projectIdOrPath), "issues", issueIid, "closed_by");
    return (response.readEntity(new GenericType<List<MergeRequest>>() { }));
}

public Pager<MergeRequest> getClosedByMergeRequests(Object projectIdOrPath, Integer issueIid, int itemsPerPage) throws GitLabApiException {
    return new Pager<MergeRequest>(this, MergeRequest.class, itemsPerPage, null,
            "projects", getProjectIdOrPath(projectIdOrPath), "issues", issueIid, "closed_by");
}
```

Reply:
Gotcha. I actually saw `Object projectIdOrPath` in other classes. But IssuesApi was still using projectId, so I went with that. I will update it.